.gitattributes

```diff
@@ -19,6 +19,7 @@
 *.mts text eol=lf whitespace=blank-at-eol,-blank-at-eof,-space-before-tab,tab-in-indent,tabwidth=2
 
 *.lockb binary diff=lockb
+bun.lock linguist-language=json
```

Comment:
I added as temporary measure as it makes the files look better (wait until https://github.com/github-linguist/linguist/pull/7193 and then _can_ remove)